doc/040_backup.rst

```diff
@@ -146,6 +146,16 @@ change detection rule based on file metadata to determine whether a file is
 likely unchanged since a previous backup. If it is, the file is not scanned
 again.
 
+The previous backup, called parent snaphot in restic terminology, is
+determined as follows. By default restic groups snapshots by hostname and backup
+paths, and then selects the latest snapshot which matches the current backup.
+You can change the selection criteria using the `--group-by`` option, which
+defaults to ``host,paths``. To select the latest snapshot with the same paths
+independent of the hostname, use ``paths``. Or to only consider the hostname
+and tags, use ``host,tags``. Alternatively, it is possible to manually specify
+a specific snapshot using the ``--parent snapshotID`` option. The option should
+also be set to the same value for the ``forget`` command.
+
```

Comment:
Some minor changes, but I suggest changing this *and the above* paragraph to the following:

~~~
When restic encounters a file that has already been backed up, whether in the
current backup or a previous one, it makes sure the file's contents is only
stored once in the repository. To do so, it normally has to scan the entire
contents of the file. Because this can be very expensive, restic also uses a
change detection rule based on file metadata to determine whether a file is
likely unchanged since a previous backup. If it is, the file is not scanned
again.

The previous backup snapshot, called "parent" snaphot in restic terminology,
is determined as follows. By default restic groups snapshots by hostname and
backup paths, and then selects the latest snapshot in the group that matches
the current backup. You can change the selection criteria using the
``--group-by`` option, which defaults to ``host,paths``. To select the latest
snapshot with the same paths independent of the hostname, use ``paths``. Or,
to only consider the hostname and tags, use ``host,tags``. Alternatively, it
is possible to manually specify a specific parent snapshot using the
``--parent`` option. Finally, Note that one would normally set the
``--group-by`` option for the ``forget`` command to the same value.
~~~

Reply:
I've change the first paragraph to use "file's content is"